mongodb-atlas-sql-interface.md

```diff
@@ -82,13 +85,6 @@ To connect using the Atlas SQL interface:
       Use the MongoDB URI obtained [in the prerequisites](#obtaining-connection-information-for-your-federated-database-instance).  Make sure that it doesn't contain your username and password. URIs containing username and/or passwords are rejected.
     - Your federated **Database** name. _Required_  
       Use the name of the federated database obtained [in the prerequisites](#obtaining-connection-information-for-your-federated-database-instance).
-   - The logging level for the underlying ODBC driver. _Optional, Default value: INFO_  
```

Comment:
The native query doesn't show for PowerQuery Online?

Reply:
Discussed at standup. At this time it appears the two are different beasts.